references/openapi-mock/apiproxy/petstore-v3.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!--
+  Copyright 2024 Google LLC
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<APIProxy revision="5" name="petstore-v3">
```

Comment:
I recommend maintaining this file in its most minimal form because it is not updated by the control plane if you make edits to new revisions and export again, resulting in a drift over time.

e.g. https://github.com/apigee/devrel/blob/main/references/auth-schemes/apiproxy/auth-schemes-v0.xml

Reply:
Got it, I will keep it with just the display name and desciption.

Reply:
SGTM!